docs/docs/faq.md

```diff
@@ -20,6 +34,13 @@ designing Tanka, we felt these add more complexity for the user than they
 provide additional value. To keep Tanka as minimal as possible, these are **not
 available** and are not likely to be ever added.
 
+## What about kubecfg ?
+
+Tanka development has started at the time when kubecfg was a part of
+already-deprecated `ksonnet` project. Although these projects are similar, Tanka
+aims to provide continuity for `ksonnet` users, whereas `kubecfg`goes the way of
+a very simple jsonnet-to-kubernetes synchronization tool.
```

Comment:
Is this their official project goal? We might underrate their efforts on the project in an potentially offending way.

Reply:
Fair enough, might sound that way (although that wasn't the intent -- I meant it more in the "take jsonnet with no extra sugaring and apply it to the cluster). To be more correct, I'll use the phrase from kubecfg's `README.md`:
```
...is really just a thin Kubernetes-specific wrapper around jsonnet evaluation
```

Reply:
Fixed.